packages/devtools_shared/test/server/devtools_extensions_api_test.dart

```diff
@@ -285,68 +285,80 @@ void _verifyExpectedRuntimeExtensions(
   _verifyExtension(
     extensions[0],
     extensionPackage: driftPackage,
-    detectedFromPackage: 'my_app',
+    detectedFromPath: 'my_app',
     fromStaticContext: false,
   );
   _verifyExtension(
     extensions[1],
     extensionPackage: providerPackage,
-    detectedFromPackage: 'my_app',
+    detectedFromPath: 'my_app',
     fromStaticContext: false,
   );
   _verifyExtension(
     extensions[2],
     extensionPackage: staticExtension1Package,
-    detectedFromPackage: 'my_app',
+    detectedFromPath: 'my_app',
     fromStaticContext: false,
   );
 }
 
 void _verifyExpectedStaticExtensions(List<DevToolsExtensionConfig> extensions) {
-  expect(extensions.length, 6);
+  expect(extensions.length, 8);
   extensions.sort();
   _verifyExtension(
     extensions[0],
     extensionPackage: driftPackage,
-    detectedFromPackage: 'my_app',
+    detectedFromPath: 'my_app',
     fromStaticContext: true,
   );
   _verifyExtension(
     extensions[1],
     extensionPackage: providerPackage,
-    detectedFromPackage: 'my_app',
+    detectedFromPath: 'my_app',
     fromStaticContext: true,
   );
   _verifyExtension(
     extensions[2],
     extensionPackage: newerStaticExtension1Package,
-    detectedFromPackage: 'other_root_2',
+    detectedFromPath: 'other_root_2',
     fromStaticContext: true,
   );
   _verifyExtension(
     extensions[3],
     extensionPackage: staticExtension1Package,
-    detectedFromPackage: 'my_app',
+    detectedFromPath: 'my_app',
     fromStaticContext: true,
   );
   _verifyExtension(
     extensions[4],
     extensionPackage: staticExtension1Package,
-    detectedFromPackage: 'other_root_1',
+    detectedFromPath: 'other_root_1',
     fromStaticContext: true,
   );
   _verifyExtension(
     extensions[5],
+    extensionPackage: staticExtension1Package,
+    detectedFromPath: 'workspace_root',
+    fromStaticContext: true,
+  );
+  _verifyExtension(
+    extensions[6],
+    extensionPackage: staticExtension1Package,
+    detectedFromPath: 'workspace_root',
+    fromStaticContext: true,
+  );
```

Comment:
why is this extension detected twice? Is it because of the workspace root pubspec and the workspace member pubspec? If so can you add a comment explaining that?

Reply:
Yes that is why, added comment.